### DISCUSSION TITLE
Just want to say thank you  to djdd87 and everyone else working on this.  I was able to get notified right away with someone trying to get in my house.

### DISCUSSION THREAD

**Discussion by Neogx** [djdd87/SynoAI#173] [General]
Many thanks, man! I installed it awhile ago and used it on telegram and its worked beautifully and tonight i was working and my phone went off at night and right away i was able to react to some punk trying to get into my house

**Comment by djdd87** [comment]
Hah - that's amazing. I'm glad it helped you.
